packages/pla/pla.2.1/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+maintainer: "Leonardo Laguna Ruiz <modlfo@gmail.com>"
+synopsis: "Pla is a simple library and ppx syntax extension to create composable templates based on verbatim strings"
+homepage: "https://modlfo.github.io/pla/"
+bug-reports: "https://github.com/modlfo/pla/issues"
+license: "MIT"
+dev-repo: "git+https://github.com/modlfo/pla.git"
+
+build: [["dune" "build" "-p" name "-j" jobs]]
+
+depends: [
+  "ocaml" { >= "4.07.0" }
+  "dune" { build & >= "2.5" }
+  "ppxlib" { >= "0.22.0" }
+  "ocaml-compiler-libs"
```

Comment:
Are you really using `ocaml-compiler-libs` directly? I can only see uses of `compiler-libs` (those are two different libraries, one is a janestreet library, the other is a package included in OCaml itself)

Reply:
```suggestion
```

Reply:
It is the `ocaml-compiler-libs`. I use it to get `Location.t`. However, when linking the library I need to use the name `compiler-libs` in the dune file. I don't know why.

Reply:
That's because `ppxlib` uses `ocaml-compiler-libs` under the hood already so that makes it a transitive dependency

Reply:
Do you want to change your usages of `compiler-libs` by `ocaml-compiler-libs` in your dune files and re-release? Or do that later (the dependency would be a bit misleading here though)

Reply:
I can remove it in the next revison I make to the library.